mypy/checker.py

```diff
@@ -740,9 +740,15 @@ def is_implicit_any(t: Type) -> bool:
 
                 # Type check initialization expressions.
                 for arg in item.arguments:
-                    init = arg.initialization_statement
-                    if init:
-                        self.accept(init)
+                    if arg.initializer is not None:
+                        name = arg.variable.name()
+                        msg = 'Incompatible default for '
+                        if name.startswith('__tuple_arg_'):
+                            msg += "tuple argument no' {}".format(name[12:])
```

Comment:
Few people will understand `argument no' 1` -- just get rid of the  `no'` and make it `argument 1` please.